server/controllers/userController.ts

```diff
@@ -9,33 +9,34 @@ const logger = log4js.getLogger('[User Controller]');
  * Creates user controller with the provided services.
  * @returns {object} User controller.
  */
-const createUserController = (
-  userService: ReturnType<typeof createUserService>,
-) => {
+const UserController = (userService: ReturnType<typeof createUserService>) => {
```

Comment:
Since this is a factory function and there are many more of them in our API, probably better to adhere to the naming convention starting with a verb like the one we had before.

Reply:
It is better to revert back to `createUserController` to remain consistent with the naming convention of our factory function pattern and the rest of the backend.